packages/compartment-mapper/test-manual/policy/app.js

```diff
@@ -0,0 +1,18 @@
+/* global require Buffer module */
+const Poet = require('entropoetry');
```

Comment:
Leaving the manual test here as illustration for the review and to make it easier to play around with an example.

Not sure if it's welcome to stay in the repository, but I'm ok removing it.

Reply:
I'm happy with leaving it here, but would you consider moving it to a `demo` subdirectory instead of `test-manual`?